src/components/MoneyRequestConfirmationList.js

```diff
@@ -699,6 +698,7 @@ function MoneyRequestConfirmationList(props) {
                             titleStyle={styles.flex1}
                             disabled={didConfirm}
                             interactive={!props.isReadOnly}
+                            rightLabel={Permissions.canUseViolations() && Boolean(props.policy.requiresCategory) ? translate('common.required') : ''}
```

Comment:
Everything looks good, but I need to make sure we're setting `requiresCategory` and `requiresTag` in the policy object in Onyx

Reply:
@cead22 is that something to be addressed here or in a separate issue?

Reply:
I confirmed this is how this is supposed to be 👍

Reply:
@cead22 just following up here. I'm rewriting the testing instructions and I've set the proper requirements in old Dot to have categories and tags be required, however I'm not seeing `requiresTags` or `requiresCategories` coming through anywhere on the props from Onyx, in the policy object or otherwise. `props.policyTags.Tag.required` comes through correctly but that's it.

Reply:
Yeah that's fine, maybe we can skip QA steps that depend on those values being true for now, since the backend change for that isn't ready yet

Reply:
Alright @cead22, I updated the test cases and made a note in the QA section that links to this thread.